wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -2,141 +2,140 @@
 {% load humanize %}
 {% load static %}
 
-<section>
-  <div class="card" id="election_{{ postelection.election.slug }}">
-    {% if postelection.cancelled %}
-      {% include "elections/includes/_cancelled_election.html" with object=postelection only %}
-    {% else %}
-      <h1>
-        <span aria-hidden="true">🗳️</span>
-        {{ postelection.election.name_without_brackets }}
-        <span aria-hidden="true">🗳️</span>
-      </h1>
-      {% if not postelection.is_pcc and not postelection.is_mayoral %}
-        <h2>{% if postelection.is_london_assembly_additional %}Additional members{% else %}{{ postelection.friendly_name }}{% endif %}</h2>
-      {% endif %}
-      {% if postelection.metadata.coronavirus_message %}
-        <div style="border:1px solid red;margin:1em 0;padding:1em">
-          <strong>{{ postelection.metadata.coronavirus_message|safe }}</strong>
-        </div>
+<div class="card" id="election_{{ postelection.election.slug }}">
+  {% if postelection.cancelled %}
+    {% include "elections/includes/_cancelled_election.html" with object=postelection only %}
+  {% else %}
+    <h1>
+      <span aria-hidden="true">🗳️</span>
```

Comment:
How does this work?

Reply:
It's just the unicode emoji "[Ballot Box with Ballot](https://emojipedia.org/ballot-box-with-ballot/)" - I like using emoji as a free way to add icons - no icon font or images required!

The `aria-hidden="true"` is to stop screen readers reading the emoji out - there's no value in the icon being there as it's just visual decoration, so we don't want each heading to be read as `"Ballot box with ballot Stroud local election ballot box with ballot"`